internal/testing/e2e/in_memory.go

```diff
@@ -86,6 +87,6 @@ func start(t *testing.T, eval evaluator.Evaluator, mmf mmfService.MatchFunction)
 	cfg.Set("logging.level", *testOnlyLoggingLevel)
 	cfg.Set(telemetry.ConfigNameEnableMetrics, *testOnlyEnableMetrics)
 
-	apptest.TestApp(t, cfg, listeners, minimatch.BindService, mmfService.BindServiceFor(mmf), evaluator.BindServiceFor(eval))
+	apptest.TestApp(t, cfg, listeners, minimatch.BindService, mmfService.BindServiceFor(mmf), defaulteval.BindService)
```

Comment:
This line shouldn't be changing, for here or in cluster.  The default evaluator is no longer used for the e2e tests.

Reply:
That makes sense, thanks!

Reply:
cluster too